mbuild/packing.py

```diff
@@ -389,14 +389,16 @@ def _validate_box(box):
 
 def _packmol_error(out, err):
     """Log packmol output to files. """
-    with open('log.txt', 'w') as log_file, open('err.txt', 'w') as err_file:
```

Comment:
Does PACKMOL not write to std_err? I think either we should leave this the way it was in case newer PACKMOL versions decide to write to std_err or remove the `err` argument if we go this route, but I'm leaning towards the former.

Reply:
Oh I see now with the change that was made to `Popen` why you made this change. So are we not capturing std_err at all now? It looks like we could [reroute this to STDOUT](https://docs.python.org/2/library/subprocess.html#popen-constructor), although I'm trying to figure out exactly how this works. It would be best probably if an error occurs for the script to exit, rather than just writing that output to a file, and maybe that is the current behavior after these changes. In any case, some additional units tests will likely be required.

Reply:
I'm going to double check, but just to clarify, what we want is any error reported by packmol to end up in log.txt, right? I'll need to check the contents of the file, but I do know the unit tests that force an error in packmol are passing.

Reply:
http://termbin.com/v6su Errors do end up in log.txt

Reply:
Okay, yeah I just wanted to make sure we were still writing Packmol's error message somewhere. Inside `log.txt` is fine. Yes, I think it might be a good idea to have a unit test just to make sure the error message is there (in case an issue arises with newer Packmol versions later down the line).